node/init.go

```diff
@@ -13,10 +13,10 @@ import (
 // Init initializes the Node FileSystem Repository for the given Node Type 'tp' in the directory under 'path' with
 // default Config. Options are applied over default Config and persisted on disk.
 func Init(path string, tp Type, options ...Option) error {
-	cfg := DefaultConfig(tp)
+	cfg, sets := DefaultConfig(tp), new(settings)
```

Comment:
Not the biggest fan of `sets`. Is there a better name? Maybe even `s`.

Reply:
Don't have strong preferences about naming here. Btw, why?

Reply:
"sets" makes me initially think of https://en.wikipedia.org/wiki/Set_(mathematics)